authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

```diff
@@ -0,0 +1,65 @@
+[id="cco-mode-mint"]
```

Comment:
Mint mode overview taken from the former CCO page. The three referenced files at the end will load properly when the other PR that added them is merged.